### PR TITLE
{containerapp} `az containerapp env update`: Fix polling Url

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ upcoming
 * 'az containerapp env create': support --mi-system-assigned and --mi-user-assigned for environment create commands
 * 'az containerapp env identity': support for container app environment assign/remove/show commands
 * 'az containerapp env storage set': Support create or update managed environment storage with NFS Azure File.
+* 'az containerapp env update': Fix polling
```
Comment (Juliehzl) [inline src/containerapp/HISTORY.rst:16]: no need to add it in history

File: src/containerapp/azext_containerapp/_clients.py [hunk 2]
```diff
@@ -307,6 +307,32 @@ class StoragePreviewClient(StorageClient):
...
+            sub_id,
+            resource_group_name,
+            name,
+            cls.api_version)
+
+        r = send_raw_request(cmd.cli_ctx, "PATCH", request_url, body=json.dumps(managed_environment_envelope))
+
+        if no_wait:
+            return
+        elif r.status_code == 202:
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/_clients.py:326]: should we also fix in core cli

Comment (Greedygre) [reply src/containerapp/azext_containerapp/_clients.py:326]: yes, Will send another PR for it.

**Review by Juliehzl — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**